examples/3d-tiles/tile-3d-layer.js

```diff
@@ -87,17 +89,21 @@ export default class Tile3DLayer extends CompositeLayer {
 
     // Traverse and and request. Update _selectedTiles so that we know what to render.
     const {height, tick} = animationProps;
-    const {cameraPosition, cameraDirection, cameraUp, zoom} = viewport;
+    const {longitude, latitude, cameraDirection, cameraUp, zoom} = viewport;
 
     // TODO: remove after sse traversal working
-    const minZoom = 14;
-    const maxZoom = 21;
+    const minZoom = 16;
+    const maxZoom = 20;
     const zoomMagic = 10000; // royalexhibition
     let zoomMap = Math.max(Math.min(zoom, maxZoom), minZoom);
     zoomMap = (zoomMap - minZoom) / (maxZoom - minZoom);
     let expMap = 1 - Math.exp(-zoomMap * 6); // Use exposure tone mapping to smooth out the sensitivity in the zoom mapping
     expMap = Math.max(Math.min(1.0 - expMap, 1), 0);
     const distanceMagic = expMap * zoomMagic;
+    const heightMagic = expMap * zoomMagic + 16; // I think royal was tiled at a height of 16
```

Comment:
Is this `heightMagic` only for royal case?

Reply:
It's for everything. I can get rid of the +16 and the comment.